cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/CwmsHttpCache.java

```diff
@@ -0,0 +1,54 @@
+package mil.army.usace.hec.cwms.http.client;
+
+import okhttp3.Cache;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+final class CwmsHttpCache {
+    static final String CACHE_SIZE_PROPERTY_KEY = "cwms.http.client.cache.size";
+    static final String CACHE_DIRECTORY_PROPERTY_KEY = "cwms.http.client.cache.directory";
+    static final Path CACHE_DEFAULT_DIRECTORY = Paths.get(System.getProperty("java.io.tmpdir"))
+            .resolve("CWMS").resolve("cwms-http-client").resolve("cache");
+    static final long CACHE_DEFAULT_SIZE_MB = 100;
+    static final Cache INSTANCE = new Builder().build();
+
+    private CwmsHttpCache() {
+        throw new AssertionError("Singleton utility class, do not instantiate");
+    }
+
+
+    static Cache getInstance() {
+        return INSTANCE;
+    }
+
+    public static class Builder{
```

Comment:
inner class should be packaged scope to match outer class

Reply:
done